runtime/doc/lua-plugins.txt

```diff
@@ -0,0 +1,50 @@
+*lua-plugins.txt*  Nvim
```

Comment:
maybe should name this `runtime-plugins`

don't need a separate helpful for every builtin plugin. except major ones with a lot of surface area.